packages/rum/src/domain/startRecordBridge.ts

```diff
@@ -0,0 +1,17 @@
+import { getEventBridge } from '@datadog/browser-core'
+import type { ViewContexts } from '@datadog/browser-rum-core'
+import type { BrowserRecord } from '../types'
+
+export function startRecordBridge(viewContexts: ViewContexts) {
+  const bridge = getEventBridge<'record', BrowserRecord>()!
+
+  return {
+    addRecord: (record: BrowserRecord) => {
+      const view = viewContexts.findView()
+      if (!view) {
+        return
+      }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏How can it happen?

Reply:
It should never happen. It was to please the typecheck but I can use `viewContexts.findView()!` if you prefer?

Reply:
Yes, in case it happens, I'd prefer to see a telemetry error than to not be aware of it.